File: app/src/main/java/com/example/puesca/Model/Usuarios.java
package com.example.puesca.Model;

import java.io.Serializable;

public class Usuarios  implements Serializable {
    public int idusuarios;
    public String nombre;
    public String cedula;
    public String estado;
    public String clave;

    public Usuarios(String nombre, String cedula, String estado,String clave) {
        this.nombre = nombre;
        this.cedula = cedula;
        this.estado = estado;
        this.clave = clave;
    }

    public Usuarios() {

    }

    public String getClave() {
        return clave;
    }

    public void setClave(String clave) {
        this.clave = clave;
    }

    public int getIdusuarios() {
        return idusuarios;
    }

    public void setIdusuarios(int idusuarios) {
        this.idusuarios = idusuarios;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCedula() {
        return cedula;
    }

    public void setCedula(String cedula) {
        this.cedula = cedula;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}

File: base/Proyecto_puesca/CORE/Usuarios.php
<?php

include 'BDD/BDD.php';

class Usuarios
{
    public $idusuarios;
    public $nombre;
    public $estado;
    public $clave;
    public $cedula;

    /**
     * Usuarios constructor.
     * @param $idusuarios
     * @param $nombre
     * @param $estado
     * @param $clave
     */
    public function __construct($idusuarios, $nombre, $estado, $clave,$cedula)
    {
        $this->idusuarios = $idusuarios;
        $this->nombre = $nombre;
        $this->estado = $estado;
        $this->clave = $clave;
        $this->cedula = $cedula;
    }

    /**
     * @return mixed
     */
    public function getCedula()
    {
        return $this->cedula;
    }

    /**
     * @param mixed $cedula
     */
    public function setCedula($cedula)
    {
        $this->cedula = $cedula;
    }

    /**
     * @return mixed
     */
    public function getIdusuarios()
    {
        return $this->idusuarios;
    }

    /**
     * @param mixed $idusuarios
     */
    public function setIdusuarios($idusuarios)
    {
        $this->idusuarios = $idusuarios;
    }

    /**
     * @return mixed
     */
    public function getNombre()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $nombre
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getClave()
    {
        return $this->clave;
    }

    /**
     * @param mixed $clave
     */
    public function setClave($clave)
    {
        $this->clave = $clave;
    }



}